src/components/Modal/ModalContainer.tsx

```diff
@@ -8,23 +8,41 @@ import { Modal } from './Modal';
 export const ModalContainer: React.FC<ModalContainerProps> = ({
   onClose,
   addTodo,
+  editTodo,
+  currIDForEdit,
+  modalType,
+  initialValueForModalWithEditFeature,
 }) => {
-  const [title, handleTitle] = useInput();
-  const [description, handleDescription] = useInput();
+  const { title: initialTitle } = initialValueForModalWithEditFeature;
+  const { description: initialDescription } =
+    initialValueForModalWithEditFeature;
 
+  const [title, handleTitle] = useInput(
+    modalType === 'edit' ? initialTitle : ''
+  );
+  const [description, handleDescription] = useInput(
+    modalType === 'edit' ? initialDescription : ''
+  );
   const createTodo = () => {
     addTodo({ id: v4(), title: title, description: description });
```

Comment:
Pass only the title and description. Create id (in normal case- the id will be generated by db) in the handler.